lib/ruby-lint/constant_loader.rb

```diff
@@ -70,13 +77,37 @@ def run(ast)
     #
     def after_initialize
       @loaded = Set.new
+      @module_nesting = []
+      @registry ||= RubyLint.registry
```

Comment:
@mvidner If this is only needed for the tests then you can just stub things out using something like `loader.should_receive(:registry).and_return(...)`